docs/tutorials/debug/debug_fault_exception.md

```diff
@@ -13,7 +13,7 @@ When the system crashes due to fault exceptions, the Mbed OS fault exception han
 
 The exceptions supported on your platform depend on the specific Cortex-M core you have in your system.
 
-For example, Cortex-M0 cores (or any ARMv6M cores) do not have MemManage, BusFault and UsageFault exceptions implemented. In those cases, all exceptions are reported as HardFault exception. Please look at the **Technical Reference Manual** and **Arm Architecture Reference Manual** documents for more information on exceptions supported for the specific core you have in your system.
+For example, Cortex-M0/M0+ processors (or any ARMv6M processors) do not have MemManage, BusFault and UsageFault exceptions implemented. In those cases, all exceptions are reported as HardFault exception. For ARMv7M processors, MemManage, BusFault and UsageFault exceptions are triggered only if they are enabled in **SHCSR** (System Handler Control and State Register). Please look at the **Technical Reference Manual** and **Arm Architecture Reference Manual** documents for more information on exceptions supported for the specific core you have in your system.
```

Comment:
Query: What triggers MemManage, BusFault and UsageFault exceptions? Please clarify for active voice.